assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +359,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
+      return route.host.indexOf(defaultHost) !== 0;
```

Comment:
Oh yeah, `!== 0` includes `-1` :)

The host we're generating now is `fmt.Sprintf("%s-%s.%s", route.ServiceName, route.Namespace, shard.DNSSuffix)`, so for now, add a trailing `.` to `defaultHost`. Checking an annotation is the right way to tell, so we'll switch to that as soon as we have it available